src/RangeSlider.elm

```diff
@@ -27,15 +27,27 @@ onChange msg input =
     Html.Events.on "change" (Json.Decode.map msg input)
 
 
-sliderInputView : CommonAttributes -> ValueAttributes msg -> Json.Decode.Decoder Float -> Html msg
-sliderInputView commonAttributes valueAttributes input =
+sliderInputView : CommonAttributes -> ValueAttributes msg -> Json.Decode.Decoder Float -> Maybe String -> Html msg
```

Comment:
:thought_balloon: I wonder how this would work out if instead of using a Maybe String we used a List String?